conda_pypi/cli/install.py

```diff
@@ -93,9 +93,22 @@ def execute(args: Namespace) -> int:
 
     # Convert package strings to MatchSpec objects
     match_specs = [MatchSpec(pkg) for pkg in args.packages]
-    converter.convert_tree(match_specs)
+    changes = converter.convert_tree(match_specs)
     channel_url = converter.repo.as_uri()
 
+    packages_to_install = changes[1]
+    converted_packages = [
+        str(pkg.to_simple_match_spec())
+        for pkg in packages_to_install
+        if pkg.channel.canonical_name == channel_url
+    ]
+    if converted_packages:
+        converted_packages_dashed = "\n - ".join(converted_packages)
+        print(f"Converted packages\n - {converted_packages_dashed}\n")
+    else:
+        print("No packages converted\n")
```

Comment:
Would we ever hit this case with `conda-pypi`?  Shouldn't there at least be one conversion? Maybe this is catching the case where we have a stashed conversion?

Reply:
This is the case when the package is already installed. Maybe this message is confusing. I'm going to back this little message out.